src/connectors/uniswap/uniswap.lp.helper.ts

```diff
@@ -145,26 +147,38 @@ export class UniswapLPHelper {
 
   getPoolContract(
     pool: string,
-    wallet: providers.StaticJsonRpcProvider | Signer
+    wallet: providers.StaticJsonRpcProvider | Signer,
   ): Contract {
     return new Contract(pool, this.poolAbi, wallet);
   }
 
+  getPoolAddress(token0: Token, token1: Token, fee: uniV3.FeeAmount): string {
```

Comment:
There is already a standard to define network-specific addresses in `uniswap.yml`, so adding a custom fn isn't a good idea